src/main/java/billgates/usecases/user_join/UserJoinController.java

```diff
@@ -0,0 +1,22 @@
+package billgates.usecases.user_join;
+
+public class UserJoinController{
+
+    private UserJoinInputPort useCase;
+
+    public UserJoinController(UserJoinInputPort useCase) {
+        this.useCase = useCase;
+    }
+
+    public void setUseCase(UserJoinInputPort useCase) {
+        this.useCase = useCase;
+    }
+
+    public UserJoinInputPort getUseCase() {
+        return useCase;
+    }
+
+    public void UserJoin(UserJoinRequestModel model){
```

Comment:
A convention error here: a method name should be starting with lower case.

Reply:
In addition, I don't think GUI will pass a UserJoinRequestModel to controller. Instead, it will just pass the username and password to it.